### PR TITLE
Improved friction parameters

### PR DESCRIPTION
**Pull request by BlueAndi**
Friction added for the robot track with rubber material and the ground on cardboard with 0.65 u_static

Zumo32U4 weight incl. the standard batteries set to 200g.

### CODE REVIEW

File: webots/worlds/LineFollowerTrack.wbt [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-  translation -0.24713614078815466 -0.04863962992854465 0.013994298332013683
-  rotation -1.0564747468923541e-06 8.746699709178704e-07 0.9999999999990595 1.5880805820884731
-  name "Zumo"
-}
-Supervisor {
-  name "Supervisor"
-  controller "Supervisor"
-  supervisor TRUE
-}
+#VRML_SIM R2023b utf8
```
Comment (gabryelreyes) [inline webots/worlds/LineFollowerTrack.wbt:1]: Line endings? It should be LF as it's the standard of Webots, but I am not sure what it is right now.

File: webots/protos/Zumo32U4.proto [hunk 2]
```diff
@@ -3,9 +3,10 @@
 EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/appearances/protos/Rubber.proto"
 
 PROTO Zumo32U4 [
-  field SFVec3f    translation  0 0 0
-  field SFRotation rotation     0 0 1 0
-  field SFString   name         "robot"
+  field SFVec3f     translation     0 0 0
+  field SFRotation  rotation        0 0 1 0
+  field SFString    name            "robot"
+  field SFString    contactMaterial "default" # Contact material of the tracks
```
Comment (gabryelreyes) [inline webots/protos/Zumo32U4.proto:9]: Why make this variable? We could set it fixed to rubber.

Comment (BlueAndi) [reply webots/protos/Zumo32U4.proto:9]: If we use the robot on other worlds, the standard contact properties are set to "default". The idea was to keep that and always specifiy in the instance of the proto which material we would like to use.

File: webots/worlds/LineFollowerTrack.wbt (hunk 1 above)
Comment (BlueAndi) [reply webots/worlds/LineFollowerTrack.wbt:1]: I saved the file with webots, so now its LF. It was CRLF before.

**Review by gabryelreyes — APPROVED** [review]
LGTM. Waiting for the other reviewers.

**Review by jkerpe — APPROVED** [review]

**Review by paulgrammy — APPROVED** [review]

**Review by KaiGlaesker — APPROVED** [review]
no findings

**Pull request merged**